src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  let result;
+
+  const adder = (n) => {
+    if (n === undefined) {
```

Comment:
do you really need to check for undefined, or you're interested in whether the value is falsy?

Reply:
if it is falsy, you can just provide default value there

Reply:
I spent a lot of time with it and could not manage to pass tests without checking for undefined or argument.length. Maybe I misunderstood idk. May I have your suggestion please?

Reply:
I meant checking like `!n`
But probably I'm not aware of all the test of this task

Reply:
oh yeah thats doable, I overthought a bit. thank you!